plink/1.90b7.2/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:22.04
+LABEL maintainer="Daniel Miller (millerd15@email.chop.edu)"
+
+RUN apt update && apt install -y curl unzip
+
+WORKDIR /opt
+
+RUN curl -so temp.zip -L https://s3.amazonaws.com/plink1-assets/plink_linux_x86_64_20231211.zip \
```

Comment:
Is downloading from amazon the new downloading from a google drive?!

Reply:
They don't provide their stuff through github unfortunately. They have their binaries detailed at the top of this page: https://www.cog-genomics.org/plink/1.9/.